contrib/aws/aws-sdk-go/aws/aws.go

```diff
@@ -78,6 +79,7 @@ func (h *handlers) Complete(req *request.Request) {
 		return
 	}
 	span.SetTag(tagAWSRetryCount, req.RetryCount)
+	span.SetTag(tagAWSRequestID, req.RequestID)
```

Comment:
Can you also modify the unit tests over in aws_test.go to check for this as well?